rental_class.rb

```diff
@@ -0,0 +1,11 @@
+class Rental
+  attr_accessor :date
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    @person = person
+    book.add_rental(self)
+    person.add_rental(self)
+  end
```

Comment:
Getters and Setters for @book and @person instance variables should be present to make relationships possible🤗

```ruby
class Rental
  attr_accessor :date, :book, :person
.
.
.
end
```

Please add them. 😊